src/device/pf_pdport.c

```diff
@@ -541,8 +540,25 @@ int pf_pdport_read_ind (
             p_pos);
          ret = 0;
       }
+      else
+      {
+         LOG_INFO (
+            PNET_LOG,
+            "PDPORT(%d): PLC could not read device PD real data (index 0x%04X) "
+            "for slot %u, subslot 0x%04X\n",
+            __LINE__,
+            index,
+            slot,
+            subslot);
+      }
       break;
+
    case PF_IDX_DEV_PDEXP_DATA:
+      LOG_INFO (
```

Comment:
Can be removed if a common error message is added.

Reply:
Done